src/constants.js

```diff
@@ -187,6 +190,7 @@ export const CLIENT_NAME_CONTENT_EXPLORER = 'ContentExplorer';
 export const CLIENT_NAME_CONTENT_PREVIEW = 'ContentPreview';
 export const CLIENT_NAME_CONTENT_SIDEBAR = 'ContentSidebar';
 export const CLIENT_NAME_OPEN_WITH = 'ContentOpenWith';
+export const CLIENT_NAME_USM_ELEMENT = 'USMElement';
```

Comment:
Could we consider a different name for the exposed component? `ContentSharing` perhaps?